src/modules/rewards/components/RewardsActivitiesList.tsx

```diff
@@ -1,26 +1,29 @@
 import { FC } from 'react';
+
+import { css } from 'styled-components';
 import InfiniteScroll from 'react-infinite-scroller';
```

Comment:
Don't think we need the infinite scroll functionality anymore. This can be removed from the component and the hook can be updated as well.